CRM/Core/BAO/FinancialTrxn.php

```diff
@@ -591,37 +629,50 @@ public static function getMembershipRevenueAmount($lineItem) {
    * Create transaction for deferred revenue.
```

Comment:
Change to 'Create db entries to recognize deferred revenue appropriately in the future'